packages/commons-ui-next/package.json

```diff
@@ -0,0 +1,45 @@
+{
+	"name": "next-commons-ui",
+	"version": "0.0.0",
+	"main": "index.js",
+	"license": "MIT",
+	"scripts": {
+		"test": "jest",
+		"lint": "eslint --fix './'",
+		"clean": "rm -rf .turbo node_modules",
+		"storybook": "start-storybook --ci -p 6006",
+		"build-storybook": "build-storybook --quiet -o dist"
+	},
+	"dependencies": {
+		"next": "12.1.0",
+		"prop-types": "^15.8.1",
+		"webpack": "^5.70.0"
+	},
+	"devDependencies": {
+		"@babel/core": "^7.17.7",
+		"@babel/preset-react": "^7.16.7",
+		"@storybook/addon-actions": "^6.4.19",
+		"@storybook/addon-essentials": "^6.4.19",
+		"@storybook/addon-links": "^6.4.19",
+		"@storybook/addons": "^6.4.19",
+		"@storybook/client-api": "^6.4.19",
+		"@storybook/react": "^6.4.19",
+		"babel-loader": "^8.2.3",
+		"config": "workspace:*",
+		"eslint": "^8.11.0",
+		"eslint-config-commons-ui": "workspace:*",
+		"jest": "^27.5.1",
+		"react": "^17.0.2",
+		"react-dom": "^17.0.2",
+		"storybook-addon-material-ui5": "^1.0.0",
+		"typescript": "^4.6.2"
+	},
+	"peerDependencies": {
+		"@mui/material": "^5.5.1",
+		"react": "^16.0.0 || ^17.0.2",
```

Comment:
Let’s stick with `17`

Reply:
That causes the problem @nerdlet raised in the morning regarding `storybook-addon-material` throwing an error. I looked into it and found this package has versions of [`react-inspector`](https://github.com/react-theming/storybook-addon-material-ui/blob/version-1/package.json#L46) and [`color-picker`](https://github.com/react-theming/storybook-addon-material-ui/blob/version-1/package.json#L47) that still rely on v16 of React thus the incompatibility issue. Looking at how `storybook-addon-material` have solved it, they just [allowed both versions](https://github.com/react-theming/storybook-addon-material-ui/pull/125/files)

So far, I am not seeing any way around it yet, other than raising an issue on the library asking them to bump up the react-inspector and color-picker because I checked and saw they have versions supporting react 17 @kilemensi

Reply:
Is it an **error** (can't proceed without fixing it) or a **warning** @gertie-sheshe? Most React version issues are usually warnings and hence me saying we should just stick to `17` because it is backward-compatible with `16`.

As far as I'm aware, current version of `react-inspector` supports React 17 but `color-picker` seems like a dead project to me (hasn't received any updates in at least 2 years).

Reply:
Fixed